pkg/util/log_test.go

```diff
@@ -43,6 +43,11 @@ func (s *logSuite) TestInitLogger(c *check.C) {
 	err := InitLogger(cfg)
 	c.Assert(err, check.IsNil)
 	c.Assert(log.GetLevel(), check.Equals, zapcore.WarnLevel)
+	err = SetLogLevel("info")
+	c.Assert(err, check.IsNil)
+	c.Assert(log.GetLevel(), check.Equals, zapcore.InfoLevel)
```

Comment:
suggest adding a test that `SetLogLevel("info")` for another time to cover more code